packages/@unimodules/core/ios/UMCore.podspec

```diff
@@ -10,12 +10,13 @@ Pod::Spec.new do |s|
   s.license        = package['license']
   s.author         = package['author']
   s.homepage       = package['homepage']
-  s.platform       = :ios, "10.0"
+  s.platform       = :ios, '10.0'
   s.source         = { :git => "https://github.com/expo/expo.git" }
-  s.source_files   = 'UMCore/**/*.{h,m}'
-  s.preserve_paths = 'UMCore/**/*.{h,m}'
-  s.requires_arc   = true
 
+  if !$ExpoUseSources&.include?(package['name']) && ENV['EXPO_USE_SOURCES'].to_i == 0 && File.exist?("#{s.name}.xcframework")
```

Comment:
One thing I'm curious about is how safe using safe navigation operator (introduced in 2015 in Ruby 2.3) is. Should we note somewhere that from now we'll require Ruby 2.3 to install dependencies?

I've checked and only since July 2020 ([`1.10.0.beta.1`](https://rubygems.org/gems/cocoapods/versions/1.10.0.beta.1)) Cocoapods requires Ruby 2.3.3, so people could be running older Rubies with older CocoaPods and could potentially run into problems evaluating this.

Reply:
Maybe we could try `.try(:include?, package['name'])` since it's an extension added by ActiveSupport which CocoaPods depend on?

Reply:
Ruby 2.3.0 was released five years ago and macOS 10.15 ships with Ruby 2.6.3. On Big Sur / macOS 11.0 I see ruby 2.6.3p62.

Since developers need Catalina anyway in order to use Xcode 11+ without workarounds, I think it's pretty reasonable to expect Ruby 2.6.3 to be installed.